generators/entity/templates/src/test/java/package/web/rest/_EntityResourceIntTest.java

```diff
@@ -195,7 +195,8 @@ public void setup() {
 
     @Before
     public void initTest() {<% if (databaseType == 'mongodb' || databaseType == 'cassandra') { %>
-        <%= entityInstance %>Repository.deleteAll();<% } %>
+        <%= entityInstance %>Repository.deleteAll();<% } if (searchEngine == 'elasticsearch') { %>
+        <%= entityInstance %>SearchRepository.deleteAll();<% } %>
```

Comment:
@geraldhumphries ok, I understand now that I've read once again your comment. I will try to find a better way to handle that...